src/components/blocks/DlLabelPicker/DlLabelPicker.vue

```diff
@@ -105,11 +105,15 @@ export default defineComponent({
         hideNoData: {
             type: Boolean,
             default: false
+        },
+        defaultSelectedLabel: {
+            type: Object as PropType<DlLabelPickerItem>,
+            default: undefined
```

Comment:
change default from undefined to null please. 
Undefineds cause an issue in vue